apps/meteor/tests/e2e/page-objects/home-team.ts

```diff
@@ -30,4 +30,8 @@ export class HomeTeam {
 	get btnTeamCreate(): Locator {
 		return this.page.locator('#modal-root button:has-text("Create")');
 	}
+
+	get fieldError() {
+		return this.page.locator('.rcx-field__error');
```

Comment:
*todo*: verify a11y of the field error and possibility of being queryable via ARIA role selectors.